frame/nft/src/lib.rs

```diff
@@ -224,7 +226,9 @@ pub mod pallet {
 							Some(instances) => {
```

Comment:
would it be possible to use ValueQuery with empty map to instead of map wrapped in option and simplify handling?

Reply:
Was wondering the same myself - I'll look into it tomorrow